### PR TITLE
[zh-cn]Update cheatsheet.md

### PR DESCRIPTION
**Pull request by yanrongshi**

### CODE REVIEW

File: content/zh-cn/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -417,6 +420,9 @@ kubectl get pods --selector=app=cassandra -o \
 kubectl get configmap myconfig \
   -o jsonpath='{.data.ca\.crt}'
 
+# 检索一个 base64 编码的值，用破折号代替下划线。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/cheatsheet.md:423]:
```suggestion
# 检索一个 base64 编码的值，其中的键名包含减号而不是下划线。
```

**Comment by tengqm** [comment]
Please always **squash your PR into a single commit**.

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -68,11 +68,6 @@ kubectl config get-contexts                          # display list of contexts
 kubectl config current-context                       # display the current-context
 kubectl config use-context my-cluster-name           # set the default context to my-cluster-name
 
-kubectl config set-cluster my-cluster-name           # set a cluster entry in the kubeconfig
-
-# configure the URL to a proxy server to use for requests made by this client in the kubeconfig
-kubectl config set-cluster my-cluster-name --proxy-url=my-proxy-url
-
 # add a new user to your kubeconf that supports basic auth
 kubectl config set-credentials kubeuser/foo.kubernetes.com --username=kubeuser --password=kubepassword
```
Comment (Sea-n) [inline content/en/docs/reference/kubectl/cheatsheet.md:72]: Why this PR reverting English changes?

Comment (yanrongshi) [reply content/en/docs/reference/kubectl/cheatsheet.md:72]: It was caused by my misoperation in rebase, maybe I should establish a PR to solve this problem

**Pull request closed without merging**